app-backend/db/src/main/scala/CampaignDao.scala

```diff
@@ -179,11 +180,58 @@ object CampaignDao extends Dao[Campaign] with ObjectPermissions[Campaign] {
   def isActiveCampaign(id: UUID): ConnectionIO[Boolean] =
     query.filter(id).filter(fr"is_active = ${true}").exists
 
+  def getChildren(campaignId: UUID): ConnectionIO[List[Campaign]] =
+    query.filter(fr"parent_campaign_id = $campaignId").list
+
   def getCloneOwners(id: UUID): ConnectionIO[List[UserThin]] =
     for {
-      campaigns <- query.filter(fr"parent_campaign_id = $id").list
+      campaigns <- getChildren(id)
       userIds = campaigns.map(_.owner)
       users <- UserDao.getUsersByIds(userIds)
       userThins = users.map(u => UserThin.fromUser(u))
     } yield userThins
+
+  def getProjectMapping(
+      childCampaignId: UUID,
+      parentDateMap: Map[Timestamp, UUID]
+  ): ConnectionIO[List[(ChildAnnotationProjectId, ParentAnnotationProjectId)]] =
+    AnnotationProjectDao.listByCampaign(childCampaignId) map { childProjects =>
+      childProjects flatMap { project =>
+        project.capturedAt flatMap { parentDateMap.get(_) } map { parentId =>
+          (
+            ChildAnnotationProjectId(project.id),
+            ParentAnnotationProjectId(parentId)
+          )
+        }
+      }
+    }
+
+  def retrieveChildCampaignAnnotations(
```

Comment:
This is where all the work happens, and it makes sense to start code review from what this method does